aiohttp_devtools/runserver/main.py

```diff
@@ -28,8 +28,7 @@ class RunServer(TypedDict):
 
 
 def runserver(**config_kwargs: Any) -> RunServer:
-    """
-    Prepare app ready to run development server.
+    """Prepare app ready to run development server.
```

Comment:
Seems like there's a ton of unrelated refactoring which doesn't exactly help review things.

Reply:
Yep, sorry. Several changes are also to fix issues highlighted by the forked tests which haven't been running for the past couple of years.

Reply:
Oh, I guess I'd still want them separate for atomicity.